src/main/java/com/ft/methodearticleinternalcomponentsmapper/messaging/MessageProducingInternalComponentsMapper.java

```diff
@@ -37,7 +35,7 @@ void mapInternalComponents(EomFile methodeContent, String transactionId, Date me
             message = messageBuilder.buildMessage(
                     internalComponentsMapper.map(methodeContent, transactionId, messageTimestamp, false)
             );
-        } catch (MethodeArticleMarkedDeletedException | MethodeArticleHasNoInternalComponentsException e) {
+        } catch (MethodeArticleMarkedDeletedException e) {
```

Comment:
I'm actually surprised that MAM is doing it like this. It seems important to me, to handle all the exceptions. They are probably a bit rare, and they just surface with the whole stack trace, without breaking the service.
I'll log them appropriately and just return, so it skips producing messages (as it already did).